src/app/features/event/card/card.component.html

```diff
@@ -0,0 +1,36 @@
+<div
+    *ngIf="card"
+    [style.background-image]="'url(' + this.url + ')'"
```

Comment:
a może przeniesiemy to cudeńko do css, a nadpisywać będziemy custom variable w css?

`[style.--background-image-path]="url"`

```css
:host {
--background-image-path: 'assets/cards/card-purple.webp';
}

 div {
   background-image: url(var(--background-image-path));
 }
```

Reply:
zmienne css nie mogą przechowywać adresów url: https://dev.to/afif/what-no-one-told-you-about-css-variables-553o#2-they-cannot-store-urls
aby ta zmienna działała zmieniłem ją na:
```
:host {
  --background-image-path: url('assets/cards/card-purple.webp');
}
```
co powoduje ze html musi wyglądać tak:
```
 [style.--background-image-path]="'url(' + url + ')'"
```
I to jest zapis taki sam jak oryginal. Do tego powoduje to error w kompilacji, mówiący że dana ścieżka nie istnieje, (sprawdziłem ,modyfikowałem jest poprawna), do tego czasem czasem się pojawia, czasem nie, zupełnie nieprzewidywalnie. Proponuje więc abyśmy zostali przy pierwotnym sposobie dodania tego url.